src/panels/energy/ha-panel-energy.ts

```diff
@@ -81,6 +86,19 @@ class PanelEnergy extends LitElement {
                     collectionKey="energy_dashboard"
                   ></hui-energy-period-selector>
                 `}
+            <ha-button-menu
```

Comment:
Let's make this a button, "COMPARE DATA" maybe as label? Or this icon 
![image](https://user-images.githubusercontent.com/1444314/169127605-5ac3bcac-675f-44d6-a469-c2722f9020a9.png)